Sources/AblyChat/Version.swift

```diff
@@ -0,0 +1,16 @@
+import Ably
+
+// Update this when you release a new version
+
+// Version information
+public let version = "0.1.0"
+
+// Channel options agent string
+public let channelOptionsAgentString = "chat-ios/\(version)"
```

Comment:
Shouldn't it include "ably" as prefix? Also "ios" will be added by cocoa. WDYT @lawrence-forooghian

Reply:
I assume this is copied from the JS implementation; let's wait and see if @umair-ably adds a spec point

Reply:
yeah that's right, just took it from JS. I think we want something else here anyway hence the added todo to revisit this